src/worker.py

```
@@ -2845,6 +2894,440 @@ async def api_list_subjects(_req, env):
+        else:  # mcq / true_false / scale
+            value = str(raw_value or "").strip()
+            if q["required"] and not value:
+                return err(f'"{q["text"]}" is required')
+            if value:
+                if value not in valid_choice_ids:
+                    return err(f'"{q["text"]}" has an invalid choice')
+                to_insert.append((q["id"], value, None))
+
+    response_id = new_id()
+
+    async def run():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `run`

Add return type annotation: `None`

(ANN202)